docs/BF Labeling Guide.md

```diff
@@ -0,0 +1,167 @@
+# Bot Framework Labeling Guidelines (DRAFT)
+
+This article describes the common labels used in the Bot Framework repositories to tag issues and PRs.
+
+It also provides guidance on how to use them.
+
+## Label format
+
+In order to be able to parse the different labels in reports we use `:` to separate the categories and subcategories in a label. The general format of a label is:
+
+```bash
+Main category:Subcategory 1:Subcategory 2
+```
+
+There's no limit to the number of subcategories but normally 2 or 3 levels at most should be enough.
+
+## Label usage
+
+Some labels can be applied multiple times to the same issue and some other shouldn't. For example, an issue could be labeled as `Area:Adaptive` **and** `Area:Skills` but it should **only** have one priority label `P:0` or `P:1`.
+
+## Label colors
+
+Labels should use the same colors across repos to improve readability and in most cases, all labels within the same main category should use the same color.
+
+## Main label categories
+
+This section describes the main label categories being used:
+
+|Category| Description  | Usage  | Example
+|---|---|---|------|
+|[Area](#area)|Defines a functional area or feature of the product.| Multiple |`Area:Skills`|
+|[Priority](#priority)|The priority for the issue.| Single |`P:0`|
+|[Status](#status)|Further describes the status of the issue throughout its lifecycle.| Single |`Status:New`|
+|[Community](#community)|Used to describe community related issues.| Single |`Community:Help Wanted`|
+|[Type](#type)|Provides additional information on the issue type.| Single |`Type:Bug`|
+|[Size](#size)|Provides an estimate for the level of effort required to resolve the issue.| Single |`Size:M`|
+|[Automation](#automation) |Used to trigger GitHub actions and workflows.| Single |`Automation:No parity`|
+|[DRI](#dri)|Special set of tags used for DRI tracking and reporting.| Multiple |`Bot Services`|
+
+### Area
+
+These labels are used to map issues to a feature or functional area of the product. All the issues with `Status:Approved` should have at least one of these labels before they can be worked on.
+
+Color: All the labels in this category should use ![#1d76db](https://via.placeholder.com/15/1d76db/000000?text=+) `#1d76db`
+
+#### Subcategories for Area
+
+|Name| Description  | Example
+|---|---|------|
+|Adaptive| TODO | `Area:Adaptive` |
+|Adaptive Expressions| TODO | `Area:Adaptive Expressions` |
+|AI-LUIS| TODO | `Area:AI-LUIS` |
+|AI-QnAMaker| TODO | `Area:AI-QnAMaker` |
+|Authentication| TODO | `Area:Authentication` |
+|Custom Adapters| TODO | `Area:Custom Adapters` |
+|Docs| TODO | `Area:Docs` |
+|Functional Tests| TODO | `Area:Functional Tests` |
```

Comment:
It would be nice to have a guidance for the case of category, i saw this topic in @cwhitten's doc,  like

Functional Tests
functional tests
functional-tests

I feel ok for upper or lower case, but i feel like "space" between may make the label looks not tighten together, especially if you have multiple labels. 

if possible, i would vote use "-" to connect

Reply:
The consensus so far us to to standardize on ": " (with a spaces after the colon) for category separators and we will use sentence case for longer labels as @JonathanFingold suggested. We all agree that is better to be consistent, this is a matter of personal pref and is hard to please everyby but . Hope you understand.